tests/10-vcdm2.js

```diff
@@ -229,70 +220,58 @@ describe('Types', function() {
         await assert.rejects(endpoints.issue(require(
           './input/credential-type-unmapped-fail.json')));
       });
-      it('type property: "If more than one URL is provided, the URLs ' +
-        'MUST be interpreted as an unordered set."', async function() {
-        this.test.link = `https://w3c.github.io/vc-data-model/#types:~:text=If%20more%20than%20one%20URL%20is%20provided%2C%20the%20URLs%20MUST%20be%20interpreted%20as%20an%20unordered%20set.`;
-        //issue VC with multiple urls in type property
-        await endpoints.issue(require(
-          './input/credential-type-urls-order-1-ok.json'));
-        //issue another VC with same urls in a different order
-        await endpoints.issue(require(
-          './input/credential-type-urls-order-2-ok.json'));
-      });
+      it('If more than one value is provided, the order does not matter.',
```

Comment:
ok so the way we've been handling this is with `(property name)` so in this case, "If more than one (type) value is provided, the order does not matter."

Reply:
Fixed in [41f2a0f](https://github.com/w3c/vc-data-model-2.0-test-suite/pull/81/commits/41f2a0f4efff0212e1e19e5fe6526cd1d023256a)